### ISSUE TITLE
AWS S3

### ISSUE THREAD

**Issue by Zyles** [roxiness/routify#225]
How does this package work with static hosting on AWS S3?

The documentation says to redirect 404 traffic to the root application for it to catch the routes.

How is this solved on AWS S3?

And to make matters worse. If I have my svelte app in a subdir, how will routing work in this case?

In my scenario I have a static HTML page at index.html (non-svelte) and svelte app in /myapp/ catching the routes for /myapp/route. So I will be unable to route to the root domain.

**Comment by jakobrosenberg** [comment]
Hi @Zyles.  I don't really have much experience with S3, so you'd probably have to google how to redirect/url rewrite.

If your Svelte app is in a subfolder, you can use basepath.

**Comment by Wolfr** [comment]
S3 uses Nginx I believe, you might be able to set up the config for that to do the proper redirection for SPA, but I don't know how exactly. I ran into a related problem a while ago and saved some links. These might be of use.

* https://www.liquidweb.com/kb/redirecting-urls-using-nginx/
* https://gist.github.com/gggritso/2e129197b527cf301064
* https://medium.com/@P_Lessing/single-page-apps-on-aws-part-1-hosting-a-website-on-s3-3c9871f12

This issue not strictly related to Routify. We are trying to keep the issue log clean with things about Routify, so if you don't mind, I will close this, and I hope we helped you on your way.
